examples/openthread_posix_example/udp.c

```diff
@@ -0,0 +1,188 @@
+/*
```

Comment:
Would you be interested/does it make sense to submit this as a PR to OpenThread to add a ATSAMR21-XPRO example? /cc @jwhui